src/zui/ZUIOrganizeSidebar/index.tsx

```diff
@@ -178,45 +207,37 @@ const ZUIOrganizeSidebar = (): JSX.Element => {
               {!open && !hover && (
                 <Avatar alt="icon" src={`/api/orgs/${orgId}/avatar`} />
               )}
-
               {open && (
-                <ZUIFuture future={model.getOrganization(orgId)}>
-                  {(data) => (
-                    <>
-                      <Box
-                        sx={{
-                          alignItems: 'center',
-                          display: 'flex',
-                        }}
-                      >
-                        <Box
-                          sx={{
-                            display: 'flex',
-                            justifyContent: 'center',
-                            width: '48px',
-                          }}
-                        >
-                          {hover ? (
-                            <IconButton onClick={handleClick}>
-                              <KeyboardDoubleArrowLeftOutlined />
-                            </IconButton>
-                          ) : (
-                            <Avatar
-                              alt="icon"
-                              src={`/api/orgs/${orgId}/avatar`}
-                            />
-                          )}
-                        </Box>
-                        <Typography variant="h6">{data.title}</Typography>
-                      </Box>
-                      <Box sx={{ display: open ? 'flex' : 'none' }}>
-                        <IconButton onClick={handleExpansion}>
-                          {checked ? <ExpandLess /> : <ExpandMore />}
+                <>
```

Comment:
I would not have expected this part of the code to change at all. I would continue using the model, `ZUIFuture` etc and that would (I assume) result in this part of the code being untouched.

Reply:
I'm not sure what I did to make this happen... I would not have expected it to change either, but obviously I did something... I'll look at what I can do to make it not appear as a change in git!